package.json

```diff
@@ -1,6 +1,10 @@
 {
   "name": "root",
   "private": true,
+  "engines": {
```

Comment:
Should we share the settings for all package.json files just in case people are trying to build each one separately?

Reply:
I thought about it, but I'm afraid they'll all get out of date and all of our commands start in this package and use lerna anyways